plugins/plugin-run-script/plugin.js

```diff
@@ -25,23 +25,8 @@ function runScriptPlugin(_, {cmd, watch, output}) {
         }
         if (stdOutput.includes('\u001bc') || stdOutput.includes('\x1Bc')) {
           log('WORKER_RESET', {});
-          log('WORKER_UPDATE', {state: ['RUNNING', 'yellow']});
           stdOutput = stdOutput.replace(/\x1Bc/, '').replace(/\u001bc/, '');
         }
-        if (cmdProgram === 'tsc') {
```

Comment:
related: `WORKER_UPDATE` was no longer used, so this entire block had essentially become a no-op. Safe to remove